packages/cf-component-flex/src/Flex.js

```diff
@@ -1,16 +1,41 @@
 const React = require('react');
 const {PropTypes} = React;
+const prefixAll = require('inline-style-prefixer/static');
 
 class Flex extends React.Component {
   render() {
     let className = 'cf-flex';
 
-    if (this.props.spacing) {
-      className += ' cf-flex--' + this.props.spacing;
+    const {
+      spacing,
+      direction,
+      wrap,
+      justifyContent,
+      alignItems,
+      alignContent
+    } = this.props;
+
+    const style = {};
+
+    if (spacing) {
+      className += ' cf-flex--' + spacing;
+    } else {
+      [
+        ['flexDirection', direction],
+        ['flexWrap', wrap],
+        ['justifyContent', justifyContent],
+        ['alignItems', alignItems],
+        ['alignContent', alignContent]
+      ].forEach(pair => {
+        const [cssProp, prop] = pair;
+        if (prop) {
+          style[cssProp] = prop;
+        }
+      });
     }
 
     return (
-      <div className={className}>
+      <div className={className} style={prefixAll(style)}>
```

Comment:
When the values are rendered, devtool only tell you the value is valid instead of why it's invalid. I mean, this isn't a new pattern, it's just instead of appending a className, we are inserting inline styles because flexbox.